src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs

```diff
@@ -7,5 +7,5 @@
     Version = "2.0.0",
```

Comment:
Can we change the version to `2.1.0` while there's a change in this module?

Reply:
We don't do that yet, until we have shipped.